src/KafkaFlow.Retry.UnitTests/KafkaFlow.Retry/Surrogate/JobDetailSurrogate.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace KafkaFlow.Retry.UnitTests.KafkaFlow.Retry.Surrogate
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Quartz;
+
+    [ExcludeFromCodeCoverage]
+    internal class JobDetailSurrogate : IJobDetail
```

Comment:
Why do we need this surrogate? Can't we just mock the interface `IJobDetail`?

Reply:
no. Removed

Reply:
done